### PR TITLE
Implemented CustomOutput as CustomView

### LINKED ISSUE #126: Missing the Android version of the custom output stream
**Actual Behaviour**
Missing the Android version of the custom output stream

**Expected Behaviour**
There should be the customoutput class.

**Would you like to work on the issue?**
Yes, I would work on this,

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #126 

**Changes**: 
Implemented CustomOutput as CustomView

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

### CODE REVIEW

**Comment by jddeep** [comment]
@mariobehling @iamareebjamal Please review.

File: app/src/main/java/io/neurolab/CustomOutputView.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package io.neurolab;
+
+import android.content.Context;
+import android.widget.Scroller;
+import android.widget.TextView;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+public class CustomOutputView extends OutputStream {
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/CustomOutputView.java]: No need to extend OutputStream

**Comment by jddeep** [comment]
@iamareebjamal Please take a look. Thanks.

File: app/src/main/java/io/neurolab/CustomOutputView.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    private Context mContext;
+
+    public CustomOutputView(Context context, TextView textArea) {
+        this.textArea = textArea;
+        this.mContext = context;
+    }
+
+    public void write(int b) {
+        // redirects data to the text area
+        textArea.append(String.valueOf((char)b));
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/CustomOutputView.java]: Either directly use char or don't cast to char

**Comment by jddeep** [comment]
@iamareebjamal Can you please have a look now. Do tell me if any more changes are required. Thanks

File: app/src/main/java/io/neurolab/CustomOutputView.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package io.neurolab;
+
+import android.content.Context;
+import android.widget.Scroller;
+import android.widget.TextView;
+
+public class CustomOutputView {
+    private TextView textArea;
+    private Context mContext;
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/CustomOutputView.java]: Don't use hungarian notation

**Comment by jddeep** [comment]
@iamareebjamal Please see now.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**